packages/dev/core/src/ShadersWGSL/iblVoxelGrid.vertex.fx

```diff
@@ -8,27 +8,200 @@
 
 // This shader uses vertex pulling to determine the
 // provoked vertex and calculate the normal. Then, based on
-// the direction of teh normal, it swizzles the position to
+// the direction of the normal, it swizzles the position to
 // maximize the rasterized area.
 #ifdef VERTEX_PULLING_USE_INDEX_BUFFER
 var<storage, read> indices : array<u32>;
 #endif
 var<storage, read> position : array<f32>;
+#if NUM_BONE_INFLUENCERS > 0
+  var<storage, read> matricesIndices : array<u32>;
+  var<storage, read> matricesWeights : array<f32>;
+  uniform vp_matricesIndices_info: vec3f;
+  uniform vp_matricesWeights_info: vec3f;
+  #if NUM_BONE_INFLUENCERS > 4
+    var<storage, read> matricesIndicesExtra : array<u32>;
+    var<storage, read> matricesWeightsExtra : array<f32>;
+    uniform vp_matricesIndicesExtra_info: vec3f;
+    uniform vp_matricesWeightsExtra_info: vec3f;
+  #endif
+#endif
 
 uniform world : mat4x4f;
 uniform invWorldScale: mat4x4f;
 
 varying vNormalizedPosition : vec3f;
 flat varying f_swizzle: i32;
 
-fn readVertexPosition(index : u32)->vec3f {
-  var pos : vec3f;
-  pos.x = position[index * 3];
-  pos.y = position[index * 3 + 1];
-  pos.z = position[index * 3 + 2];
-  return pos;
+uniform vp_position_info: vec3f; // (bufferIndex, offset, stride, type)

```

Comment:
I'm not sure what **bufferIndex** refers to?

Reply:
That was an out of date comment. I've fixed it.